src/Components/Categories.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import Header from './Header';
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Categories extends Component {
+  render() {
+    return (
+      <main>
+        <Header />
+        <h2>Hello Categories</h2>
```

Comment:
- [ ] Please change this heading to a button with the text `Check status` as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/216345360-d9d68851-9085-445c-a93d-1413fb4067a5.png)